### PR TITLE
Update Slack Info

### PR DESCRIPTION
**Pull request by mostlikelee**
moving failure alerts to the P2 channel

### CODE REVIEW

File: .github/workflows/generate-cve.yml [hunk 1]
```diff
@@ -104,11 +104,11 @@ jobs:
...
                   "text": {
                     "type": "mrkdwn",
-                    "text": "Security artifacts generation result: ${{ job.status }}\nhttps://github.com/fleetdm/nvd/actions/runs/${{  github.run_id }}\n${{ github.event.pull_request.html_url || github.event.head.html_url }}"
+                    "text": "Security artifacts generation result: ${{ job.status }}\nhttps://github.com/fleetdm/vulnerabilities/actions/runs/${{  github.run_id }}\n${{ github.event.pull_request.html_url || github.event.head.html_url }}"
                   }
                 }
               ]
             }
         env:
-          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_G_HELP_P1_WEBHOOK_URL }}
```
Comment (mostlikelee) [inline .github/workflows/generate-cve.yml:113]: @lukeheath can you confirm this is the correct secret?

File: .github/workflows/generate-cve.yml [hunk 2]
```diff
@@ -104,11 +104,11 @@ jobs:
...
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_G_HELP_P2_WEBHOOK_URL }}
```
Comment (mostlikelee) [inline .github/workflows/generate-cve.yml:113]: @lukeheath can you confirm this is the correct secret?

Comment (lukeheath) [reply .github/workflows/generate-cve.yml:113]: I just added it!

**Review by lukeheath — APPROVED** [review]
❤️

**Pull request merged**